option optimize_for = LITE_RUNTIME;

message EquipDressRequest
{
 repeated uint32 pos = 1;
 repeated uint32 posZhengXing = 2;
}
